packages/hardhat-core/src/internal/hardhat-network/provider/modules/hardhat.ts

```diff
@@ -180,6 +202,76 @@ export class HardhatModule {
     return true;
   }
 
+  // hardhat_setBalance
+
+  private _setBalanceParams(params: any[]): [Buffer, BN] {
+    return validateParams(params, rpcAddress, rpcQuantity);
+  }
+
+  private async _setBalanceAction(address: Buffer, newBalance: BN) {
+    await this._node.setAccountBalance(new Address(address), newBalance);
+    return true;
+  }
+
+  // hardhat_setCode
+
+  private _setCodeParams(params: any[]): [Buffer, Buffer] {
+    return validateParams(params, rpcAddress, rpcData);
+  }
+
+  private async _setCodeAction(address: Buffer, newCode: Buffer) {
+    await this._node.setAccountCode(new Address(address), newCode);
+    return true;
+  }
+
+  // hardhat_setNonce
+
+  private _setNonceParams(params: any[]): [Buffer, BN] {
+    return validateParams(params, rpcAddress, rpcQuantity);
+  }
+
+  private async _setNonceAction(address: Buffer, newNonce: BN) {
+    await this._node.setAccountNonce(new Address(address), newNonce);
+    return true;
+  }
+
+  // hardhat_setStorageSlot
+
+  private _setStorageSlotParams(params: any[]): [Buffer, BN, Buffer] {
+    const [b, slotIndex, value] = validateParams(
```

Comment:
Let's use `address` instead of `b`. (Sorry, I should've suggested this in the initial comment, but I didn't realize that the first variable had to be extracted too for the return value).

Reply:
Addressed in https://github.com/nomiclabs/hardhat/pull/1435/commits/79467c56d0ba37208027aaff8ee92e1f2d88623c